vault/core.go

```diff
@@ -696,6 +696,9 @@ type Core struct {
 	// if populated, the callback is called for every request
 	// for testing purposes
 	requestResponseCallback func(logical.Backend, *logical.Request, *logical.Response)
+
+	// If no role-based lease count quota or role based rate limiting quota is enabled, don't track lease counts by role
```

Comment:
I like the succinct version you have in https://github.com/hashicorp/vault/pull/22651/files#diff-278ba6b34b839340763b2dc9d4278acf4d4b5826d5d41b13dc81d584e5d446bcR71

Should we use that same blurb here and below?

Reply:
yes good call! much clearer